tests/system_info.c

```diff
@@ -219,7 +219,11 @@ static void print_oqs_configuration(void) {
 	 * USE_SANITIZER: -fsanitize= option present in compile options
 	 * OQS_ENABLE_TEST_CONSTANT_TIME: only shown below
 	 */
-	printf("OQS version:      %s\n", OQS_VERSION_TEXT);
+#if defined(OQS_VERSION_PRE_RELEASE)
```

Comment:
Is the idea to completely drop this define when doing a release then? Or if just setting an empty pre-release string is that equal to`!defined`? Maybe worth while adding proposed wording for the changes to the [Release wiki](https://github.com/open-quantum-safe/liboqs/wiki/Release-process#code-change) in the comment?

Reply:
I've updated the release wiki to explain what's going on here. I don't have an opinion one way or the other whether it is better for `OQS_VERSION_PRE_RELEASE` to be defined-but-empty or undefined for releases.

Reply:
I'd think it'd be easier to a consumer of `liboqs` could do `ifndef OQS_VERSION_PRE_RELEASE` to detect/distinguish (using) a release.

Reply:
Okay -- which is what's currently implemented in the code. So can I mark this as resolved?